R/print.data.table.R

```diff
@@ -9,7 +9,8 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
                trunc.cols=getOption("datatable.print.trunc.cols"),
                quote=FALSE,
                na.print=NULL,
-               timezone=FALSE, ...) {
+               timezone=FALSE,
+               indices=FALSE, ...) {
```

Comment:
looks somewhat inconsistent to have `datatable.print.keys` option but not `datatable.print.indices`, should we make this controlled by an option?

Reply:
sure! should i rename the argument to `print.indices` for consistency as well?

Reply:
changed to print.indices and added as a new option